libs/fluxc/src/main/java/org/wordpress/android/fluxc/model/SiteModel.java

```diff
@@ -1110,7 +1110,7 @@ public void setPublishedStatus(int publishedStatus) {
     }
 
     public Boolean getCanBlaze() {
-        return mCanBlaze;
+        return mCanBlaze == true;
```

Comment:
This is not enough for the fix, we just moved the NPE exception here instead of `IsBlazeEnabled`. 

When unboxing from the `Boolean` type to the primitive type, we'll get an NPE, and to reproduce it just comment the `this.mCanBlaze = canBlaze;` in the line 1117 to make sure we keep the default value (simulating what would happen if `options` is missing from `/me/sites` response), the app will crash with the following exception:

```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
                                                                                                    	at org.wordpress.android.fluxc.model.SiteModel.getCanBlaze(SiteModel.java:1113)
                                                                                                    	at com.woocommerce.android.ui.blaze.IsBlazeEnabled.invoke(IsBlazeEnabled.kt:25)
```

If we want to keep the `Boolean` class as type, we need to improve this check to something like:

```suggestion
        return mCanBlaze != null && mCanBlaze;
```